Model/Gallery.php

```diff
@@ -43,7 +43,7 @@
     /**
      * @var string
      */
-    protected $defaultFormat;
+    protected $defaultFormat = 'reference';
```

Comment:
Souldn't it also be defined as a default with doctrine ? http://stackoverflow.com/questions/3376881/default-value-in-doctrine#3379560